lib/workers/global/limits.ts

```diff
@@ -0,0 +1,40 @@
+import { logger } from '../../logger';
+
+const limitsToInit = ['prCommitsPerRunLimit'];
+const l = {};
+const v = {};
+
+export function init() {
+  let i;
+  for (i = 0; i < limitsToInit.length; i += 1) {
+    if (process.env[limitsToInit[i]]) {
+      setLimit(limitsToInit[i], parseInt(process.env[limitsToInit[i]], 10));
```

Comment:
What about treating it the same way the `autodiscoverFilter` is treated as this too is a self hosted global option.

Reply:
Here we should not use `process.env`. Use passed `config`

Reply:
I was not sure about how to treat this global parameter. But I think you are right about treating the same way as `autodiscoverFilter`

Reply:
I changed it so it used definitions.ts